include/tf2_2d/rotation_impl.hpp

```diff
@@ -31,17 +31,18 @@
  *  ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
  *  POSSIBILITY OF SUCH DAMAGE.
  */
-#ifndef TF2_2D_ROTATION_IMPL_H
-#define TF2_2D_ROTATION_IMPL_H
-
-#include <tf2/LinearMath/Scalar.h>  // NOLINT: The tf2 MinMax.h file does not include all requirements. Consequently,
-#include <tf2/LinearMath/MinMax.h>  // NOLINT: the order of the headers here is important.
-#include <tf2_2d/vector2.h>
+#ifndef TF2_2D__ROTATION_IMPL_HPP_
+#define TF2_2D__ROTATION_IMPL_HPP_
 
 #include <Eigen/Core>
 
+// TODO(sloretz) remove Scalar.h when ros2/geometry2#559 is merged
```

Comment:
Reminder here to change when https://github.com/ros2/geometry2/pull/559 is merged

- https://github.com/ros2/geometry2/pull/559

Reply:
Done in 6e611a6